app/uk/gov/hmrc/tariffclassificationfrontend/connector/CasesConnector.scala

```diff
@@ -30,6 +30,11 @@ import scala.concurrent.Future
 @Singleton
 class CasesConnector @Inject()(configuration: AppConfig, client: HttpClient) {
```

Comment:
I would write for now this:
```
  def getOne(reference: String)(implicit hc: HeaderCarrier): Future[Option[Case]] = {
    val url = s"${configuration.bindingTariffClassificationUrl}/cases/$reference"
    client.GET[Option[Case]](url)
  }

  def getGatewayCases(implicit hc: HeaderCarrier): Future[Seq[Case]] = {
    val url = s"${configuration.bindingTariffClassificationUrl}/cases?queue_id=none&assignee_id=none&sort-by=elapsed-days"
    client.GET[Seq[Case]](url)
  }
```

Reply:
Will do, It's probably the Java developer in me that prefers the `+`